_config.yml

```diff
@@ -1,7 +1,13 @@
+plugins:
+  - jekyll-paginate
```

Comment:
Getting this error when building now:

```
run into trouble, you can find helpful resources at https://jekyllrb.com/help/! 
5:27:05 PM: jekyll 3.9.4 | Error:  jekyll-paginate
5:27:05 PM: /opt/buildhome/.rvm/gems/ruby-3.3.2/gems/jekyll-3.9.4/lib/jekyll/external.rb:68:in `rescue in block in require_with_graceful_fail': jekyll-paginate (Jekyll::Errors::MissingDependencyException)
```

I have never used a Jekyll plugin before and am not clear on how to approach it. How difficult would it be to do pagination without the plugin?

Reply:
I can write it up in JS, but I think that making use of the built-in solution might make sense here, we probably won't need it to do anything fancier than the bare bones pagination with control that it offers.
It should be possible to install it on the docker image like this:
```
 command: gem install jekyll-paginate && jekyll serve --watch --force_polling --verbose --livereload
 ```

Reply:
Or perhaps we could use "infinite" scrolling to load more posts with lazy method instead of pagination? https://jekyllcodex.org/without-plugin/lazy-loading/